mkat_fpga_tests/test_cbf.py

```diff
@@ -561,10 +563,10 @@ def test_sfdr_peaks(self):
         # Channel responses higher than -cutoff dB relative to expected channel
         extra_peaks = []
 
-        start_chan = 2048 # skip DC channel since dsim puts out zeros
-        # TODO NM 2015-09-04 Need to check this for all channels?
+        # Checking for all channels.
```

Comment:
The comment about skipping the DC channel should be retained -- that is the reason we don't start at channel 0.